src/services/swap_service.ts

```diff
@@ -233,6 +255,57 @@ export class SwapService {
             .filter(p => p) as GetTokenPricesResponse;
         return prices;
     }
+    private async _getSwapQuoteForWethAsync(
+        params: CalculateSwapQuoteParams,
+        isUnwrap: boolean,
+    ): Promise<GetSwapQuoteResponse> {
+        const {
+            from,
+            buyTokenAddress,
+            sellTokenAddress,
+            buyAmount,
+            sellAmount,
+            affiliateAddress,
+            gasPrice: providedGasPrice,
+        } = params;
+        const amount = buyAmount || sellAmount;
+        if (amount === undefined) {
+            throw new Error('sellAmount or buyAmount required');
+        }
+        const data = (isUnwrap
+            ? this._wethContract.withdraw(amount)
+            : this._wethContract.deposit()
+        ).getABIEncodedTransactionData();
+        const value = isUnwrap ? ZERO : amount;
+        const affiliatedData = this._attributeCallData(data, affiliateAddress);
+        const gasPrice = providedGasPrice || (await this._protocolFeeUtils.getGasPriceEstimationOrThrowAsync());
```

Comment:
This can be a TODO but I don't think its worth an aggressive 8 gwei gas price for wrap/unwrap.